src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -79,6 +82,7 @@
 
 public class ShopifySdk {
```

Comment:
Can you add some more unit tests to increase code coverage above 80%? Currently the quality gate is failing.